docs/getting-started/user-guides/using-paymetheus.md

```diff
@@ -0,0 +1,151 @@
+
+Don't worry if you didn't understand that quote. What it means is that you create
+an address that can be accessed by two wallets. Only one wallet needs to be available
+to use the address. This means that the pool can vote on your behalf and you can vote
+using your own wallet if the pool stops working.  
+
+It DOES NOT give the pool access to your funds. All you are doing is granting voting
+rights to the wallet. This is why you provide a ticket address so the network knows
+where to send your funds to after the ticket votes/expires. The pool does not touch
+your funds.  
+
+It is recommended that you generate a new account when joining a stake pool. This is
+because accounts are hardened so in the case of a total stake pool
+failure/shutdown, it would be safe to give the private key to another stake pool
+as long as that account only does voting and nothing else.  
+
+Official stake pools are [listed here](/mining/proof-of-stake#sign-up-for-a-stake-pool).
+All stake pools run the same basic code, but may differ in the amount of redundancy available.
+More redundancy equals less chance of missed votes (although all pools will have missed votes
+as many missed votes are caused by PoW miners). In order to ensure one pool does't become too large,
+it is recommended that you join a smaller pool. This further minimizes the (VERY) small chance that a pool
+owner could exert influence over the way their pool votes if it's too large. Decred already contains checks
+for this, but by spreading tickets around pools, it makes the network even less decentralized.
+
+![Creating voting account](/img/Paymetheus-create-voting-account.png)  
+
+To generate a public key address, use the "Request Payment" view to generate a
+new address for the voting account.  The pubkey address (below the shorter P2PKH
+address) will be given to the pool.
+
+<i class="fa fa-exclamation-triangle"></i> **On mainnet, the address will begin with the letters Ds.**
+
+![Generate pubkey address](/img/Paymetheus-generate-pubkey-address.png)  
+
+<i class="fa fa-exclamation-triangle"></i> **On mainnet, the public key address will begin with the letters Dk.**
+
+Copy and paste the public key address into the stake pool's submit address form
+and click the submit button.  Your multisig script and P2SH address will be
+generated and you will be redirected to the tickets page.
+
+Next, import the pool's 1-of-2 multisignature script (on the Scripts tab).  This allows the funds to
+be recovered in case of pool failure.
+
+![Importing a script](/img/Paymetheus-import-script.png)  
+
+Finally, tickets can be purchased.  Select the "Using a pool?" checkbox and copy
+the values from the pool.  When all fields are set and if there is enough input
+value is available, the purchase button will become enabled and tickets may be
+purchased.
+
+NOTE: While you can purchase tickets using Paymetheus, it cannot vote for you so at this time you must use a pool.
```

Comment:
I realize you have links just below this to the solo voting instructions but this needs to be reworded, you are NOT required to use a pool if you choose to run your own voting wallet(s).